node/src/components/linear_chain.rs

```diff
@@ -407,9 +427,20 @@ where
                     era_id,
                     ..
                 } = *fs;
-                if let Err(err) = fs.verify() {
-                    warn!(%block_hash, %public_key, %err, "received invalid finality signature");
-                    return Effects::new();
+                if let Some(last_block) = self.latest_block.as_ref() {
```

Comment:
```suggestion
                if let Some(latest_block) = self.latest_block.as_ref() {
```
(Or `self.last_block`?)

Reply:
Renamed in [`31bccd8` (#1144)](https://github.com/CasperLabs/casper-node/pull/1144/commits/31bccd8eacaf40767367af900ea6e25019463f0f)